content/en/security_platform/cspm/resource_catalog.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Datadog Resource Catalog
+kind: documentation
+is_beta: true
+private: true
+further_reading:
+- link: "/security_platform/cspm/custom_rules/schema/"
+  tag: "Documentation"
+  text: "Cloud Resource Schema"
+---
+
+{{< beta-callout url="https://google.com" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
+  Resource Catalog is currently in private beta. Fill out this form if you would like to access it.
+{{< /beta-callout >}}
+
+{{< img src="security_platform/cspm/resource_catalog/resource_catalog.png" alt="Resource Catalog map view displaying host and cloud resources grouped by category and region." style="width:100%;" >}}
+
+## Overview
+
+Datadog Resource Catalog provides a high-level overview of the hosts and resources in your cloud and hybrid environments. View information such as tags, configuration details, relationship between assets, misconfigurations, and threats. See what team is responsible for each resource, and what security findings have been found. Access dashboards and Datadog views that receive and monitor telemetry and security data for each resource.
+
+The Resource Catalog leverages existing Datadog cloud integrations and the Datadog agent and includes cloud resources such as databases, storage services, and hosts.
+
+**Note**: In order for cloud resources to appear in the Resource Catalog, you must first enable [Cloud Security Management][1] for each cloud account.
+
+The Resource Catalog is useful for:
+
+- Training new site reliability and security engineers by providing a clear view of all resources, their relationships, and their security posture, along with information about teams responsible for them and services running on them.
+- Speeding incident recovery by increasing confidence and simplifying locating owners of upstream and downstream resources.
+- Ensuring proper security coverage by identifying resources that are most prone to misconfigurations or are not actively reporting security findings.
+- Facilitating the practice of good tagging to optimize cross-telemetry insights.
+- Providing engineering leadership with a high-level view of security practices across teams and cloud accounts.
+
+## Browse the Resource Catalog
+
+On the [Resource Catalog page][2], explore the cloud resources in your Datadog organization which are either detected from an Agent on a host, or from configuring a cloud integration on the resource. Information about the resources in your organization is organized into two views: List and Map.
```

Comment:
```suggestion
On the [Resource Catalog page][2], explore the cloud resources in your Datadog organization. The catalog detects a resource either because it has an Agent installed on it, or because a cloud integration is configured on it. Information about the resources in your organization is shown on two views: List and Map.
```